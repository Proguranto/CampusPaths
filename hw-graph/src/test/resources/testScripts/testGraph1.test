# Tests acyclic graph creation/viewing of a graph
# with 1 node

CreateGraph graph1

AddNode graph1 n1

ListNodes graph1
