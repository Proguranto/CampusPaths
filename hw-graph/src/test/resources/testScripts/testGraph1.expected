# Tests acyclic graph creation/viewing of a graph
# with 1 node

created graph1

added node n1 to graph1

graph1 contains: n1